src/extras/usdImagingLite/engine.h

```diff
@@ -0,0 +1,188 @@
+/**********************************************************************
+* Copyright 2020 Advanced Micro Devices, Inc
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+********************************************************************/
+#ifndef PXR_USD_IMAGING_USD_IMAGING_LITE_ENGINE_H
+#define PXR_USD_IMAGING_USD_IMAGING_LITE_ENGINE_H
+
+#include "pxr/pxr.h"
+#include "pxr/usd/usd/stage.h"
+
+#include "pxr/imaging/hd/engine.h"
+#include "pxr/imaging/hd/rendererPlugin.h"
+#include "pxr/imaging/hdx/taskController.h"
+#include "pxr/usdImaging/usdImaging/delegate.h"
+
+#include "pxr/usdImaging/usdImagingLite/api.h"
+#include "pxr/usdImaging/usdImagingLite/renderParams.h"
+#include "pxr/usdImaging/usdImagingLite/renderDataDelegate.h"
+#include "pxr/usdImaging/usdImagingLite/renderTask.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+USDIMAGINGLITE_API
+int render(UsdPrim root, const UsdImagingLiteRenderParams &params);
+
+
+/// \class UsdImagingLiteEngine
+///
+/// The UsdImagingGLEngine is the main entry point API for rendering USD scenes.
```

Comment:
Please add some comments (and change name in comment) about what this does different